foundations/foundation-vertx/src/main/java/org/apache/servicecomb/foundation/vertx/http/HttpServletResponseEx.java

```diff
@@ -48,4 +48,8 @@ default Flowable<Buffer> getFlowableBuffer() {
   }
 
   void endResponse() throws IOException;
+
+  default void setChunkedForEvent(boolean chunked) {
+    // not set header transfer-encoding=chunked in Rest Over Servlet, or will have Multiple in response.
+  }
```

Comment:
Maybe change `setChunked` implementation to first check if header exists and then add it?

Reply:
add checked in setChunked method still have multiple headers of HttpServletResponseEx, code like this:

default void setChunked(boolean chunked) {
    if (getHeader(HttpHeaders.TRANSFER_ENCODING.toString()) != null) {
      return;
    }
    setHeader(HttpHeaders.TRANSFER_ENCODING.toString(), HttpHeaders.CHUNKED.toString());
  }